images/kubevirt-user-guide/Dockerfile

```diff
@@ -1,33 +1,32 @@
 # BASE
-FROM registry.fedoraproject.org/fedora:34
+FROM registry.fedoraproject.org/fedora:36
 
+# COPY app configs
+WORKDIR /src
 ENV LC_ALL=en_US.UTF-8
+COPY ./_config/src/ ./
```

Comment:
The presubmit test build looks to be failing here:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2333/build-kubevirt-user-guide-image/1571809333564411904#1:build-log.txt%3A60

Reply:
Not sure why this is happening. If someone knows, please give us some insights. I just updated the Dockerfile using the previous method for getting the Gemfile (`mkdir /src && cd /src && curll https://raw.../Gemfile`).

Reply:
I think this was seen with the kubevirt.github.io image - the presubmit and postsubmits may have to updated similiar to this  - https://github.com/kubevirt/project-infra/pull/2266

Reply:
Thanks for the clarification. Could copying this [line](https://github.com/cwilkers/project-infra/blob/16116f56d73d004964d96df3464d0c25013088fe/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L195) behind this [line](https://github.com/cwilkers/project-infra/blob/16116f56d73d004964d96df3464d0c25013088fe/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L222) fix the issue? Same for [postsubmits](https://github.com/cwilkers/project-infra/blob/16116f56d73d004964d96df3464d0c25013088fe/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml) in the proper lines.

Reply:
I can try in a separate PR. However, I'm not sure how to test it locally :S
Nevertheless, with the last modification I've posted yesterday the test seems to work now.

Reply:
I've removed that line and restored `mkdir -p /src && cd /src && \
  curl https://raw.githubusercontent.com/kubevirt/user-guide/main/_config/src/Gemfile -o Gemfile && \`.